docs/translations/translation-management.md

```diff
@@ -0,0 +1,68 @@
+# Translation management
```

Comment:
I've added it, so that people can see how to contribute to translations. WordPress.org states: 

> [...] The initial language pack for the plugin will be generated when 90% of the [Stable (latest release)](https://translate.wordpress.org/projects/wp-plugins/woo-gutenberg-products-block/stable/de/default/) sub-project strings have been translated [...].

Currently, only 11 locals have a translation status of >= 90%. I do agree, that this might not look relevant for developers. That said, we also have a design section, which is less relevant for developers, but it is relevant for designers. Similar to the design section, I believe that the `# Translation management` section is useful for `Translation Contributors`.

Reply:
I see, so something like this document?

https://github.com/woocommerce/woocommerce/wiki/Translating-WooCommerce

Reply:
> I see, so something like this document?

Yes. That's what I had in mind, just with a bit more context and concrete steps on our end.